scripts/download-sentry-cli.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+cd $(dirname "$0")
+REPO=getsentry/sentry-cli
+VERSION=1.70.0
```

Comment:
There's already 1.71.0: https://github.com/getsentry/sentry-cli/releases

```suggestion
VERSION=1.71.0
```

But we don't need to update right now, nothing on that release we need here.